client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -233,11 +237,16 @@ class ImageEditorCanvas extends Component {
 			canvasX = -50 * widthRatio - 100 * leftRatio,
 			canvasY = -50 * heightRatio - 100 * topRatio;
 
+		const canvasOffsetTop = canvas.offsetTop;
+		const canvasOffsetLeft = canvas.offsetLeft;
+		const canvasOffsetWidth = canvas.offsetWidth;
+		const canvasOffsetHeight = canvas.offsetHeight;
+
```

Comment:
what's the purpose of creating these constants? if it's only to be more clear I could suggest destructing the canvas object:

```es6
const { offsetTop, offsetLeft, offsetWidth, offsetHeight } = canvas;
```

... then 

```es6

offsetTop - offsetHeight * -canvasY / 100,
offsetLeft - offsetWidth * -canvasX / 100,
offsetTop + offsetHeight * ( 1 + canvasY / 100 ),
offsetLeft + offsetWidth * ( 1 + canvasX / 100 )
```